Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/ImporterCommands.cpp

```diff
@@ -108,6 +108,12 @@ namespace CommandSystem
             GetCommandManager()->ExecuteCommandInsideCommand(AZStd::string::format("Select -actorID %i", actor->GetID()).c_str(), outResult);
         }
 
+        // Automatically set the motion extraction node if one is not provided.
+        if (!actor->GetMotionExtractionNode())
+        {
+            actor->AutoSetMotionExtractionNode();
+        }
```

Comment:
We should do this in the actor itself, elsewise the behavior will be different in the main editor vs the animation editor, as in the main editor the import commands are not used. Can you change that?